docs/consume-packages/Central-Package-Management.md

```diff
@@ -211,5 +211,3 @@ There are 3 package sources defined in your configuration. When using central pa
 
 > [!Note]
 > Central package management is in active development. We appreciate you trying it out and providing any feedback you may have at [NuGet/Home](https://github.com/nuget/home/issues).
->
```

Comment:
Should we include a version min VS or .NET CLI for CPM instead? Otherwise, someone going to try on unsupported version.

Reply:
When reading this document from the start, it's my understanding that lines 21-32 describe the supported version information. So I think repeating it here would be unnecessary repetition. I'm happy to add a link to that section if that is desired to get this PR merged. https://github.com/NuGet/docs.microsoft.com-nuget/blob/7d69328361120711c529380134e15771d246beae/docs/consume-packages/Central-Package-Management.md?plain=1#L21-L32